packages/test/test-end-to-end-tests/src/test/batching.spec.ts

```diff
@@ -600,7 +600,7 @@ describeNoCompat("Flushing ops", (getTestObjectProvider) => {
 
 	describe("Batch validation when using getPendingLocalState()", () => {
 		beforeEach(async () => {
-			await setupContainers({ enableOfflineLoad: true });
+			await setupContainers();
```

Comment:
does this test still test the same thing? can you validate the exception, as now i think closeAndGetPendingLocalState fails due to offline being disabled